hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -692,9 +687,17 @@ public static SpatialCurveCompositionStrategyType fromValue(String value) {
   /**
    * Layout optimization strategies such as Z-order/Hilbert space-curves, etc
    */
+  @EnumDescription("Determines ordering strategy for records layout optimization")
   public enum LayoutOptimizationStrategy {
```

Comment:
I don't update the actual property because it requires "linear, z-order or hilbert" instead of the enum names LINEAR, ZORDER, HILBERT

Reply:
Do you think approach like `ErrorWriteFailureStrategy` could be helpful here?